### DISCUSSION TITLE
[Feature] improve multi-select in search result view

### DISCUSSION THREAD

**Discussion by aisbergde** [immich-app/immich#6162] [Feature Request]
### The feature

In the search result view, each asset needs to be selected individually. I suggest improving it, with features, similar to the timeline view:
- allow shift+left click to mark one or more ranges
- may, allow to "select all"

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Closing in favour of #9159, which is more general.
